api4/post.go

```diff
@@ -330,26 +330,42 @@ func searchPosts(c *Context, w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	includeDeletedChannels := r.URL.Query().Get("include_deleted_channels") == "true"
```

Comment:
This won't be backwards compatibly since we're changing how this parameter is specified. We should try falling back to the query parameter if it's not specified in the body

Reply:
Yes. But this feature was never released. (it's experimental)

Reply:
@crspeller Are we removing a feature from the `ExperimentalViewArchivedChannels` setting?

Reply:
No.